client/public/index.html

```diff
@@ -14,7 +14,7 @@
 	  <link href="./styles/nav-bar.css" rel="stylesheet"></link>
 	  <link href="./styles/splash-container.css" rel="stylesheet"></link>
 	  <link href="./styles/features.css" rel="stylesheet"></link>
-	  <link href="../lib/prism/prism.css" rel="stylesheet"></link>
+	  <link href="./styles/product-select.css" rel="stylesheet"></link>	  
 	  <link href="../../node_modules/materialize-css/dist/materialize.min.css" rel="stylesheet"></link>
```

Comment:
We can't include styles from `node_modules` as they aren't available via routing. This is intentional.